client/v2/algod/accountInformation.go

```diff
@@ -0,0 +1,18 @@
+package algod
+
+import (
+	"context"
+	"fmt"
+	"github.com/algorand/go-algorand-sdk/client/v2/common"
+	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+)
+
+type AccountInformation struct {
+	c       *Client
+	account string
+}
+
+func (s *AccountInformation) Do(ctx context.Context, headers ...*common.Header) (result models.Account, err error) {
```

Comment:
I'm a little confused by the `common.Header` pattern. These endpoints have different parameters, how do you deal with the fact that `/v2/accounts` doesn't support filtering by a txid but `/v2/transactions` does?

Reply:
The server will return an error, so we're not dead in the water, but if it was organized differently those sorts of errors could be caught by the compiler.

Reply:
nevermind! I mistook headers and parameters